src/NuGet.Core/NuGet.Protocol.Core.v3/LegacyFeed/V2FeedListResourceProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Protocol.Core.Types;
+using NuGet.Protocol.LocalRepositories;
+
+namespace NuGet.Protocol.LegacyFeed
```

Comment:
Please check all of your other new types for this

---
In reply to: [94993986](https://github.com/NuGet/NuGet.Client/pull/1088#discussion_r94993986) [](ancestors = 94993986)